### ISSUE TITLE
Illegal option pipefail

### ISSUE THREAD

**Issue by fredZen** [frno7/psgplay#11]
I hit a little bump building psgplay. Here's the end of the output:

```
$ make ALSA=1
...
  CC      lib/m68k/m68kdasm.o
  CC      lib/m68k/softfloat.o
  CC      lib/psgplay/psgplay.o
  CC      lib/psgplay/sndh.o
script/version: 4: set: Illegal option -o pipefail
lib/psgplay/Makefile:15: recipe for target 'lib/psgplay/version.c' failed
make: *** [lib/psgplay/version.c] Error 2
```

I discovered that `set -o pipefail` is a bashism, and that `bash` is not the default shell on my machine.

I was able to fix the problem by changing the shebang of `script/version` from `#!/bin/sh` to `#!/bin/bash`

Happy to submit a PR if you like.

**Comment by frno7** [comment]
A patch submission would be lovely, thanks @fredZen! I suppose you have PSG play working with ALSA now?

**Comment by frno7** [comment]
Fixed in commit b44f0f33a5dc8c20a421151c729d664ba6f9b6ad. Thank you for your report, and please let me know if there are any other issues.
